### PR TITLE
Add CSM acceptance test suite

### PR DESCRIPTION
**Pull request by skhomuti**
fixes:
- execute a vote before acceptance tests
- cache vote state using snapshot for better test execution time

### CODE REVIEW

**Review by dgusakov — APPROVED** [review]

File: archive/scripts/vote_sr_v2_holesky.py [hunk 1]
```diff
@@ -358,8 +358,8 @@ def start_vote(tx_params: Dict[str, str], silent: bool) -> Tuple[int, Optional[T
             agent_forward(
                 [
                     (
-                        contracts.csmHashConsensus.address,
-                        contracts.csmHashConsensus.updateInitialEpoch.encode_input(CS_ORACLE_INITIAL_EPOCH),
+                        contracts.csm_hash_consensus.address,
```
Comment (mkurayan) [inline archive/scripts/vote_sr_v2_holesky.py:361]: Not sure that we need support archived scripts, as far as I know they mostly kept for history only.

File: tests/acceptance/conftest.py [hunk 2]
```diff
@@ -12,9 +12,17 @@
 ENV_REPORT_AFTER_VOTE = "REPORT_AFTER_VOTE"
 ENV_FILL_SIMPLE_DVT = "FILL_SIMPLE_DVT"
 
+snapshot_id = None
```
Comment (mkurayan) [inline tests/acceptance/conftest.py:15]: Please make sure that this compatible with Ganache

File: tests/acceptance/conftest.py [hunk 3]
```diff
@@ -12,9 +12,17 @@
...
+
 
 @pytest.fixture(scope="function", autouse=is_there_any_vote_scripts() or is_there_any_upgrade_scripts())
 def autoexecute_vote(helpers, vote_ids_from_env, accounts, stranger):
+    global snapshot_id
```
Comment (mkurayan) [inline tests/acceptance/conftest.py:20]: Please make sure that this compatible with Ganache

File: utils/import_current_votes.py [hunk 4]
```diff
@@ -5,12 +5,19 @@
 from brownie import accounts
 from brownie.network.transaction import TransactionReceipt
 from utils.test.helpers import ETH
-from utils.config import LDO_HOLDER_ADDRESS_FOR_TESTS, get_is_live, contracts
+from utils.config import LDO_HOLDER_ADDRESS_FOR_TESTS, get_is_live, contracts, network_name
+
+
+def _get_network():
```
Comment (mkurayan) [inline utils/import_current_votes.py:11]: After discussion we decided to keep current convention, the vote script will  be moved to the root folder, those changes do not needed.

File: utils/import_current_votes.py [hunk 5]
```diff
@@ -5,7 +5,7 @@
 from brownie import accounts
 from brownie.network.transaction import TransactionReceipt
 from utils.test.helpers import ETH
-from utils.config import LDO_HOLDER_ADDRESS_FOR_TESTS, get_is_live, contracts
+from utils.config import LDO_HOLDER_ADDRESS_FOR_TESTS, get_is_live, contracts, network_name
```
Comment (mkurayan) [inline utils/import_current_votes.py:8]: Looks like network_name not used.

**Review by mkurayan — APPROVED** [review]

**Pull request merged**